packages/shared/src/components/widgets/SocialShare.tsx

```diff
@@ -62,93 +72,103 @@ export const SocialShare = ({
       }),
     );
 
-  const onShareToSquad = (squad: Squad) => {
-    trackEvent(postAnalyticsEvent(AnalyticsEvent.StartShareToSquad, post));
-    openModal({
-      type: LazyModal.CreateSharedPost,
-      props: {
-        squad,
-        preview: post,
-        onSharedSuccessfully: () => {
-          trackEvent(postAnalyticsEvent(AnalyticsEvent.ShareToSquad, post));
-          return onSquadShare;
-        },
-      },
-    });
+  const { onSubmitPost, isPosting } = usePostToSquad({
+    initialPreview: post,
+    onPostSuccess: () => {
+      trackEvent(postAnalyticsEvent(AnalyticsEvent.ShareToSquad, post));
```

Comment:
With this did we now loose the "StartShareToSquad" tracking or is it moved somewhere?
Maybe it's just not needed anymore, but good to know for analytics

Reply:
Let me verify quickly.

Reply:
The "StartShareToSquad" is used when opening the share comment modal. The "SocialShare" component is already opened, so we have tracked it already somewhere else in this case.

Reply:
Hmm ok, I'll double check with Ido later, but good for now, thanks for checking this.